ern-runner-gen-android/test/fixtures/simple-android-runner/app/build.gradle

```diff
@@ -60,5 +60,5 @@ android {
 
 dependencies {
     // Recompile this dependency on every build as it can be updated at any time (snapshot)
-    api('com.walmartlabs.ern:runner-ern-container:1.0.0') { changing = true }
+    api('com.walmartlabs.ern:myminiappa-runner-ern-container:1.0.0') { changing = true }
```

Comment:
If the applicationId is `com.walmartlabs.ern.dummy`, should this identifier be `myminiappa` or `dummy` here?